build.gradle.kts

```diff
@@ -202,7 +202,8 @@ jsonSchema2Pojo {
 
 node {
     download.set(true)
-    version.set("18.13.0")
+    // try to keep this (LTS) Node version as much in sync with the version used in our package.json as possible
```

Comment:
I don't see `engines` section in any of our package.json, so do we need this comment?

Reply:
Maybe not. I guess I was referring to `"@types/node": "^20.11.20",`. I will remove the comment for now.